### PR TITLE
Binary foreign key enhancement #13894

### PR DESCRIPTION
**Pull request by williamdes**
Patch for binary foreign key insert/edit dropdown bug
Fixes: https://github.com/phpmyadmin/phpmyadmin/issues/13894

### CODE REVIEW

File: libraries/classes/Relation.php [hunk 1]
```diff
@@ -1199,7 +1200,7 @@ public function purgeHistory($username)
      *
      * @access  protected
      */
-    public function buildForeignDropdown(array $foreign, $data, $mode)
+    public function buildForeignDropdown($column, array $foreign, $data, $mode)
```
Comment (mauriciofauth) [inline libraries/classes/Relation.php]: I don't know if it's necessary to create this `$column` parameter.

Is it possible to do something like this:
https://github.com/phpmyadmin/phpmyadmin/blob/d921ab0abb7c2b9989cd54ab8143413cd74ba422/libraries/classes/Display/Results.php#L5127-L5146

**Comment by williamdes** [comment]
@mauriciofauth please re-review when you have time :)

File: libraries/classes/Relation.php (hunk 1 above)
Comment (williamdes) [reply libraries/classes/Relation.php]: @mauriciofauth done ! :)

**Comment by williamdes** [comment]
@mauriciofauth can you please review this PR, I made the changes you requested :+1: 
Sorry for asking, but this PR patches an issue opened in 12/2017.
I want to contribute on other issues as soon as this one is closed :)

**Comment by williamdes** [comment]
@mauriciofauth [IMPORTANT] can you re-open this PR, I introduced an error.
Try to search using "Search" in this found/random database : https://demo.phpmyadmin.net/master-config/tbl_select.php?db=CAMPEONATO&table=arbitros 

Do I create (another another issue & PR) or (a commit to this PR) for adding the ability to search hex data ?

**Comment by mauriciofauth** [comment]
@williamdes You can open a new issue, as it's a new issue, even though they are related.

**Comment by williamdes** [comment]
@mauriciofauth I will do that.
I added a commit to this branch to fix my error.

**Pull request merged**